valhalla/jawn/src/managers/score/ScoreManager.ts

```diff
@@ -72,10 +81,15 @@ export class ScoreManager extends BaseManager {
       }
 
       const requestInClickhouse = await this.scoreStore.putScoresIntoClickhouse(
```

Comment:
I dont see anywhere this is putting the message into Kafka

Reply:
I think we can change this in follow PR, because nobody use Scores API now, but 100% we need to send to kafka as well

Reply:
ok moved this to kafka too